test/common/testlib.py

```diff
@@ -784,9 +788,6 @@ def skipImage(reason, *args):
     return lambda func: func
 
 class Policy(object):
-    def __init__(self):
-        self.retryable = True
```

Comment:
I interpret this as an actual policy, like the class says. This variable wouldn't change to `False` after exceeding a number of tries, for example. Like @stefwalter said, we could influence this via user flags (`opts.thorough`) or depending on other criteria.